android-components/components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/MenuButton.kt

```diff
@@ -92,6 +92,13 @@ internal class MenuButton(
 
     fun setColorFilter(@ColorInt color: Int) = impl.setColorFilter(color)
 
+    /**
+     * Hides the menu button.
+     */
+    fun setShouldBeHidden(shouldBeHidden: Boolean) {
```

Comment:
Let's add an @param for `shouldBeHidden`

Reply:
Added!